src/main/scala/beam/agentsim/infrastructure/geozone/TopDownEqualDemandSplitter.scala

```diff
@@ -2,15 +2,14 @@ package beam.agentsim.infrastructure.geozone
 
 object TopDownEqualDemandSplitter {
 
-  val MaxResolution = 12
-
   private[geozone] def chooseOneToSplit(
     children: IndexedSeq[Hexagon[_]],
     bucketsGoal: Int,
-    numberOfPoints: Int
+    numberOfPoints: Int,
```

Comment:
where is min resolution being set?

Reply:
Example in the line of code pointed by the link below:
https://github.com/LBNL-UCB-STI/beam/pull/2763/files#diff-01fd57ea1dc0392e0c7f3c9e1bd70736R20